.github/workflows/verify-pr-commit.yml

```diff
@@ -463,15 +460,14 @@ jobs:
     needs: build-binaries
     name: Verify Docker Images
     runs-on: ubuntu-20.04
-    container: ubuntu:20.04
+    container: public.ecr.aws/lts/ubuntu:20.04
     steps:
       - name: Install Required Packages
         run: |
           apt-get update
           apt-get install -y curl
           curl -fsSL https://get.docker.com -o get-docker.sh
           sh get-docker.sh
-          docker run hello-world
```

Comment:
Was this removal intended?

Reply:
It looks like just removing a check that docker is working.  @demisx ?